src/sketchup-stl.rb

```diff
@@ -52,7 +52,7 @@ def self.translate(string)
       'This is an open source project sponsored by the SketchUp team. More ' <<
       'info and updates at https://github.com/SketchUp/sketchup-stl'
     )
-    extension.version = '2.1.6'
```

Comment:
Tags in your repo doesn't follow the Pull Request - I think. The tag is something I have to set.